src/screens/SurveyDetail/question.js

```diff
@@ -1,22 +1,28 @@
-import React from 'react';
+import React, { useContext, useState } from 'react';
 
-import SurveySelectField from 'components/SurveySelectField';
-import SurveyTextAreaField from 'components/SurveyTextAreaField';
-import SurveyTextField from 'components/SurveyTextField';
+import { DetailsContext } from 'contexts/details';
 import closeIcon from 'images/close-button-white.svg';
 import nextIcon from 'images/next-button-black.svg';
+import DetermineQuestionType from 'screens/SurveyDetail/QuestionBuilders/determineQuestionType';
+import filterQuestionList from 'screens/SurveyDetail/QuestionBuilders/filterQuestionList';
+import questionProperties from 'screens/SurveyDetail/QuestionBuilders/questionProperties';
 
 const Questions = () => {
-  const question = 'How fulfilled did you feel during this WFH period?';
-  const options = [
-    { name: 'Bangladesh', value: 'BD' },
-    { name: 'India', value: 'IN' },
-    { name: 'Nepal', value: 'NP' },
-    { name: 'Bhutan', value: 'BH' },
-    { name: 'Thailand', value: 'TH' },
-    { name: 'Singapore', value: 'SG' },
-    { name: 'Vietnam', value: 'VN' }
-  ];
+  const detailsContext = useContext(DetailsContext);
+  const filteredQuestions = filterQuestionList(detailsContext.questionList);
+  const [currentQuestionProperties, setCurrentQuestionProperties] = useState(
```

Comment:
How about renaming this so it become shorter 😄 
```suggestion
  const [currentQuestion, setCurrentQuestion] = useState(
```